src/api/api.cmd

```diff
@@ -11,11 +11,11 @@
 :: restore
 :: build
 :: pack
-msbuild -p:Configuration=%_C% api.proj -v:m -m -nr:false || exit /b
+msbuild -p:Configuration=%_C% api.proj -m -nr:false -v:m -nologo || exit /b
 
 :: test
-dotnet test -c %_C% --no-build burn\test\WixToolsetTest.Mba.Core\WixToolsetTest.Mba.Core.csproj || exit /b
-dotnet test -c %_C% --no-build wix\api_wix.sln || exit /b
+dotnet test -c %_C% --no-build --nologo -v m burn\test\WixToolsetTest.Mba.Core\WixToolsetTest.Mba.Core.csproj || exit /b
+dotnet test -c %_C% --no-build --nologo -v m wix\api_wix.sln || exit /b
```

Comment:
I know you already put this in some places earlier, but why do we want minimal verbosity? If something goes wrong, I don't want to have to run another build to find out why.

Reply:
Error messages still show up. `-v m` just hides the spew from compiler invocations and such.

In a dream world, there's a single MSBuild invocation for the whole world so you could easily specify additional switches (e.g., `/bl`).

Reply:
I'm still planning to add /bl before I'm done with this PR. I find those are the most enjoyable way to diagnose build failures.

Reply:
I am definitely not looking forward to downloading binary logs to find out why the build failed on the CI server.

Reply:
They're smaller than diag-level text logs -- I want binlogs for Burn!